plugins/woocommerce/changelog/61678-fix-retrieving-3-letters-country-from-3-letters-with-package

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Attempts to identify the PayPal two-letter country code from a three-letter code using the `symfony/intl` composer package, and applies minor additional fixes
```

Comment:
This changelog entry is no longer accurate after the changing the dependency to `league/iso3166`. Same goes for the PR description.
```suggestion
Attempts to identify the PayPal two-letter country code from a three-letter code using the `league/iso3166` composer package, and applies minor additional fixes
```